config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -1570,3 +1570,47 @@ presubmits:
           limits:
             cpu: 4
             memory: 6Gi
+
+  - name: pull-kubernetes-node-kubelet-serial-pod-disruption-conditions
+    cluster: k8s-infra-prow-build
+    always_run: false
+    optional: true
+    skip_report: false
+    skip_branches:
+      - release-\d+\.\d+  # per-release image
+    annotations:
+      testgrid-dashboards: sig-node-presubmits
+      testgrid-tab-name: pr-kubelet-gce-e2e-pod-disruption-conditions
+      description: Executes pod disruption conditions e2e node tests
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    spec:
+      containers:
+        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20221024-d0c013ee2d-master
+          resources:
+            limits:
+              cpu: 2
+              memory: 2Gi
+            requests:
+              cpu: 2
+              memory: 2Gi
+          args:
+            - --root=/go/src
+            - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+            - "--upload=gs://kubernetes-jenkins/pr-logs"
+            - --timeout=240
+            - --scenario=kubernetes_e2e
+            - --
+            - --deployment=node
+            - --gcp-zone=us-west1-b
+            - --node-test-args=--feature-gates=PodDisruptionConditions=true --kubelet-flags="--cgroups-per-qos=true --cgroup-root=/"
+            - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/image-config-serial.yaml
+
+            - --node-tests=true
+            - --provider=gce
+            - --test_args=--nodes=1 --skip="" --focus="\[NodeFeature:PodDisruptionConditions\]"
```

Comment:
do these tests needs to be removed from any other jobs?

Reply:
This is part of the eviction tests (https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/eviction_test.go#L505) so it won't run anywhere other then the eviction tests (which are currently failing).

Reply:
Indeed, being able to run the PodDisruptionCondition test (which is based on the Eviction test, which appears stable) is the main goal as the Evictions tests are currently disabled. Still, this newly added test for PodDisruptionConditions during shutdown: https://github.com/kubernetes/kubernetes/blob/70e73a6cdaab5dae41357e3a25d6ac60aff62438/test/e2e_node/node_shutdown_linux_test.go#L59
would run on both configurations as currently it runs as part of this configuration (along with other graceful node shutdown tests):
https://testgrid.k8s.io/sig-node-presubmits#pr-node-kubelet-serial-containerd
I think the new tests are rather fast and it should not be a problem there is an overlap. Similarly, the CPU manager tests run as part of the above mentioned configuration even though they have a dedicated pull configuration.